### PR TITLE
Update Alerts styles

### LINKED ISSUE #7003: Update caseflow alerts to match uswds
- Remove alert overwrites. 
- update no-title option to use alerts slim

### PR DESCRIPTION
**Pull request by nolawi**
Resolves #7003

### Description
Please explain the changes you made here.

<details><summary>Before</summary>

screen shot 2018-09-19 at 11 02 27 am

</details>

<details><summary>After</summary>

screen shot 2018-09-19 at 11 00 44 am

</details>

### CODE REVIEW

File: client/app/reader/SideBarDocumentInformation.jsx [hunk 1]
```diff
@@ -24,8 +24,8 @@ class SideBarDocumentInformation extends PureComponent {
 
     if (this.props.didLoadAppealFail) {
       renderComponent = <Alert
-        title="Unable to retrieve claim details"
         type="error">
+        Unable to retrieve claim details <br />
```
Comment (nolawi) [inline client/app/reader/SideBarDocumentInformation.jsx]: using no-title option here

File: client/app/styles/_main.scss [hunk 2]
```diff
@@ -319,36 +319,6 @@ select {
   }
 }
 
-.usa-alert {
-  &.lower-margin {
-    // This value is copied from USWDS.
-    margin-bottom: 1.5em;
```
Comment (nolawi) [inline client/app/styles/_main.scss]: default `$spacing-medium` almost same size

File: client/app/styles/app.scss [hunk 3]
```diff
@@ -13,6 +13,7 @@
 @import 'reader/_reader_styles_root.scss';
 @import 'hearings/_hearings_styles_root.scss';
 @import '_style_guide.scss';
+@import '_alerts.scss';
```
Comment (mdbenjam) [inline client/app/styles/app.scss]: So now that we're including stuff on the frontend. I think we should move these imports to the component that uses them. It's clearer than having one giant include file.

Comment (mdbenjam) [reply client/app/styles/app.scss]: Or even convert it to use glamor and be in the component itself.

**Pull request merged**